hacking/test-module

```diff
@@ -152,6 +153,8 @@ def boilerplate_module(modfile, args, interpreters, check, destfile):
         task_vars=task_vars
     )
 
+    module_data = to_text(module_data)
```

Comment:
It's probably okay to do it like this since we have a coding convention of using utf-8 text but I would probably keep as a byte string and modify the other pieces of code to use a byte string.  The reason is that we're going to write this data out to disk soon and data going to disk needs to be bytes.  So there's less work to do if we keep it as bytes to begin with.

Keeping it as bytes, the other things to change are:

``` python
if module_style == 'new' and b'ANSIBALLZ_WRAPPER = True' in module_data:
```
and
``` python
modifle2 = open(modfile2_path, 'wb')
```